public/style.css

```diff
@@ -47189,7 +47189,7 @@ input[type="text"]{
   transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
 }
 
-input:focus, select:focus{
+input:focus, select:focus, textarea:focus-visible{
   border-width: 1px;
   border-style: solid;
   outline: 2px solid transparent;
```

Comment:
looks like the outline will be 2px black here, can it has the same outline style as other fields